components/host-dashboard/HostDashboardExpenses.js

```diff
@@ -170,14 +179,30 @@ const HostDashboardExpenses = ({ hostSlug, isNewAdmin }) => {
 
   return (
     <Box maxWidth={1000} m="0 auto" px={2}>
-      {!loading && data?.host.hasDisputedOrders && (
+      {!loading && (data?.host.hasDisputedOrders || data?.host.hasInReviewOrders) && (
         <Box mb={4}>
           <MessageBox type="warning" withIcon>
-            <FormattedMessage
-              id="host.disputes.warning"
-              defaultMessage="Fraud Warning: There are disputed charges that need review."
-            />{' '}
-            <Link href={`/${hostSlug}/admin/orders?status=DISPUTED`}>Review Disputes</Link>
+            <List>
+              {data?.host.hasDisputedOrders && (
+                <ListItem paddingTop={10} paddingBottom={10}>
+                  <FormattedMessage
+                    id="host.disputes.warning"
+                    defaultMessage="Fraud Warning: There are disputed charges that need review."
+                  />{' '}
+                  <Link href={`/${hostSlug}/admin/orders?status=DISPUTED`}>Disputed Orders</Link>{' '}
+                </ListItem>
+              )}
+
+              {data?.host.hasInReviewOrders && (
+                <ListItem paddingTop={10} paddingBottom={10}>
+                  <FormattedMessage
+                    id="host.in_review.warning"
+                    defaultMessage="Fraud Warning: There are charges under review that need attention."
```

Comment:
I feel like _Fraud Warning_ suggests that there were, in fact, frauds, which in some cases may be misleading. Should we use something more neutral? Something like _Fraud Protection Warning_ perhaps?

Reply:
I agree with that!